stacks-signer/src/chainstate.rs

```diff
@@ -56,6 +84,26 @@ pub struct SortitionState {
     pub consensus_hash: ConsensusHash,
     /// what is this signer's view of the this sortition's miner? did they misbehave?
     pub miner_status: SortitionMinerStatus,
+    /// the timestamp in the burn block that performed this sortition
+    pub burn_header_timestamp: u64,
+    /// the burn header hash of the burn block that performed this sortition
+    pub burn_block_hash: BurnchainHeaderHash,
+}
+
+/// Captures the configuration settings used by the signer when evaluating block proposals.
+#[derive(Debug, Clone)]
```

Comment:
Not a big deal but it seems a bit odd to have a config with a single value. Do we forsee more config options being added to this for proposal evaluation? Otherwise I might remove and store first_proposal_burn_block_timing directly in SortitionsView 

Just a preference though.